packages/did-core-test-server/suites/did-spec/default.json

```diff
@@ -2,53 +2,59 @@
           "application/did+json",
           "application/did+ld+json"
         ],
-        "dids": ["did:example:123"],
-        "didParameters": {
-          "hl": "did:example:123?hl=zQmWvQxTqbG2Z9HPJgG57jjwR154cKhbtJenbyYTWkjgF3e",
-          "service": "did:example:123?service=agent&relativeRef=%2Fpath%2Fto%2Fresource",
-          "relative-ref": "did:example:123?service=agent&relativeRef=%2Fpath%2Fto%2Fresource",
-          "version-id": "did:example:123?version-id=0.1.0",
-          "version-time": "did:example:123?version-time=1601151242"
-        },
-        "did:example:123": {
+        "dids": ["did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"],
+        "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB": {
           "application/did+json": {
             "didDocument": {
-              "@context": ["https://www.w3.org/ns/did/v1"],
-              "id": "did:example:123",
-              "canonicalId": "did:example:one-two-three",
-              "equivalentId": "did:example:one-two-three",
+              "@context": [
+                "https://www.w3.org/ns/did/v1",
+                {
+                  "@base": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
+                }
+              ],
+              "id": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
               "verificationMethod": [
                 {
-                  "id": "#key-0",
+                  "id": "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
                   "type": "JsonWebKey2020",
-                  "controller": "did:example:123",
+                  "controller": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
                   "publicKeyJwk": {
-                    "kty": "OKP",
                     "crv": "Ed25519",
-                    "x": "VDXDwuGKVq91zxU6q7__jLDUq8_C5cuxECgd-1feFTE"
+                    "x": "LKacPea7et-QMhcyS02ZjsTHEVadsCj_XNnfFJYbwHo",
+                    "kty": "OKP"
                   }
                 },
                 {
-                  "id": "#key-1",
+                  "id": "#z6LSh8yKFPF2bLnD9mHQZMJL6zQQZsowYUeUaDd39rMLHBBx",
                   "type": "JsonWebKey2020",
-                  "controller": "did:example:123",
+                  "controller": "did:key:z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB",
                   "publicKeyJwk": {
                     "kty": "OKP",
                     "crv": "X25519",
-                    "x": "3kY9jl1by7pLzgJktUH-e9H6fihdVUb00-sTzkfmIl8"
+                    "x": "USe317eZOgxr3-neAHaJ8fZ056kb7miDxZvb5ymFKVM"
                   }
                 }
               ],
-              "authentication": ["#key-0"],
-              "assertionMethod": ["#key-0"],
-              "capabilityInvocation": ["#key-0"],
-              "capabilityDelegation": ["#key-0"],
-              "keyAgreement": ["#key-1"]
+              "authentication": [
+                "#z6MkhTZLPqg7kYwBe6y8utZZr1NfMViqXWep7rY7ogUSPgsB"
```

Comment:
Again, why not fully expanded?

Reply:
Our implementation prefers relative IRIs always, they are equivalent when compared via n-quads.

Reply:
The real reason is that we want a did-key representation that looks like sidetree based dids, for testing... similar to the use of did-key with did:v1 I imagine.

As long as our implementation and yours produce the same n-quads we consider the implementations to be equivalent.

We also write tests using VCs and resolvers to show that they implementations both work, even though the look different.